### PR TITLE
Added CDN url to the frontend config

### LINKED ISSUE #5637: MapCard previews by ID are not using the CDN url

### PR DESCRIPTION
**Pull request by ethervoid**
Closes #5637  

* Added the CDN url to the frontend config to be used when we request the static map previews images

### CODE REVIEW

**Comment by ethervoid** [comment]
Please @rafatower check the backend part of this PR

**Comment by javierarce** [comment]
Can you have a look, @xavijam?

File: lib/assets/javascripts/cartodb/common/views/mapcard_preview.js [hunk 1]
```diff
@@ -40,9 +45,24 @@ module.exports = cdb.core.View.extend({
...
+      protocol: protocol,
+      username: this.options.username,
+      account_host: this.options.account_host,
+      tpl: this._generateImageTemplate(),
+      width: this.width,
+      height: this.height
+    };
+
+    if (this.options.cdn_config) {
+      options = _.extend(options, { cdn: this.options.cdn_config[protocol] });
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/views/mapcard_preview.js]: Do you know that we can require the cdb variable with ```cdb.config.get('cdb_url')``` wherever you need?

**Comment by javierarce** [comment]
Better?

File: lib/assets/javascripts/cartodb/common/views/mapcard_preview.js [hunk 2]
```diff
@@ -40,9 +45,25 @@ module.exports = cdb.core.View.extend({
   },
 
   _loadFromVisId: function() {
-
     var protocol = this._isHTTPS() ? 'https': 'http';
-    var url = protocol + '://' + this.options.username + '.' + this.options.account_host + '/api/v1/map/static/named/' + this._generateImageTemplate() + '/' + this.width + '/' + this.height + '.png';
+    var cdnConfig = cdb.config.get('cdn_url');
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/views/mapcard_preview.js]: :clap: working great?

**Comment by xavijam** [comment]
For me perfect :+1:, works in each new page?

File: lib/assets/javascripts/cartodb/common/views/mapcard_preview.js (hunk 2 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/views/mapcard_preview.js]: Yeps :)

**Comment by javierarce** [comment]
Yes, I added the config in the entry points of user_feed and explore.

**Comment by xavijam** [comment]
:es:

**Comment by rafatower** [comment]
ok to backend part.

sorry about the late response

**Comment by xavijam** [comment]
:clap:

**Pull request merged**